Sources/CIColor+Extensions.swift

```diff
@@ -0,0 +1,63 @@
+import CoreImage
+
+
+public extension CIColor {
+
+    /// Initializes a `CIColor` object with the specified `white` component value in all three (RGB) channels.
+    /// - Parameters:
+    ///   - white: The unpremultiplied component value that should be use for all three color channels.
+    ///   - alpha: The alpha (opacity) component of the color.
+    ///   - colorSpace: The color space in which to create the new color. This color space must conform to the `CGColorSpaceModel.rgb` color space model.
+    convenience init?(white: CGFloat, alpha: CGFloat = 1.0, colorSpace: CGColorSpace? = nil) {
+        if let colorSpace = colorSpace {
+            self.init(red: white, green: white, blue: white, alpha: alpha, colorSpace: colorSpace)
+        } else {
+            self.init(red: white, green: white, blue: white, alpha: alpha)
+        }
+    }
+
+    /// Initializes a `CIColor` object with the specified extended white component value in all three (RGB) channels.
+    ///
+    /// The color will use the extended linear sRGB color space, which allows EDR values outside of the `[0...1]` SDR range.
+    ///
+    /// - Parameters:
+    ///   - white: The unpremultiplied component value that should be use for all three color channels.
+    ///            This value can be of the `[0...1]` SDR range to create an EDR color.
+    ///   - alpha: The alpha (opacity) component of the color.
+    convenience init?(extendedWhite white: CGFloat, alpha: CGFloat = 1.0) {
```

Comment:
This is only a convenience method that inits the color with a specific color space.
I probably could add checks to the above API, but then I'd need to manipulate the passed color space there. And I can't do the same for the RGB initializers since they are built-in. That's why I decided for more explicit initializers here.